src/app/pages/categories/pages-categories.module.ts

```diff
@@ -18,6 +21,7 @@ const routes: Routes = [
   declarations: [
     PagesCategoriesManagementComponent,
     PagesCategoriesEditorComponent,
+    TransactionCategoryDeletingModalComponent,
```

Comment:
In that case component should be renamed to `PagesCategoriesCategoryDeletingModalComponent`